app/services/step_by_step_page_reverter.rb

```diff
@@ -52,4 +53,12 @@ def steps
   def logic(step)
     step[:logic] || "number"
   end
+
+  def contents(step_contents)
+    contents_list = step_contents.map do |content|
+      content[:text]
+    end
+
+    contents_list.join("\r\n")
```

Comment:
I'm interested in this new line.  I guess it works?  Does `\n` work too?

Reply:
@sihugh I looked at how steps are stored in the database and just copied that!

Reply:
Actually @sihugh, I think I need to join on a double newline, so that paragraphs and lists have an empty line between them.